File: utils/is_line_equal.go
package utils

func IsLineEqual(line1 []string, line2 []string) bool {
	if len(line1) != len(line2) {
		return false
	}

	for i, v := range line1 {
		if v != line2[i] {
			return false
		}
	}

	return true
}

File: utils/isLineEqual.ts
export function isLineEqual(line1: string[], line2: string[]): boolean {
  if (line1.length !== line2.length) {
    return false;
  }

  for (let i = 0; i < line1.length; i++) {
    if (line1[i] !== line2[i]) {
      return false;
    }
  }

  return true;
}
